uefi-sct/SctPkg/TestCase/UEFI/EFI/BootServices/ProtocolHandlerServices/BlackBoxTest/ProtocolHandlerBBTestStress.c

```diff
@@ -1813,7 +1815,12 @@ BBTestCombinationTest2CheckPoint4 (
     AssertionType = EFI_TEST_ASSERTION_PASSED;
   } else {
     AssertionType = EFI_TEST_ASSERTION_FAILED;
+  } 
+  
+  if (Index == TPL_ARRAY_SIZE){
+    Index -= 1;
   }
+

```

Comment:
I don't see the relation between TPL level and this OpenProtocol test, so I would suggest removing the line 1834 "TplArray[Index]" (to make StandardLib->RecordAssertion not print the TPL info). In this case, we don't have to add code  here to fix the issue.